t/25-cache-service.t

```diff
@@ -118,8 +142,11 @@ sub test_download {
 
     ok !$cache_client->enqueue($asset_request), "enqueued id $id, asset $asset";
 
+    my $worker = $t->app->minion->worker->register;    # doesn't help
```

Comment:
That line and the next `unregister` line below were leftovers from my testing, they are not needed.

Reply:
hm, this line and the `unregister` line is still in there, just that you removed the comment